### PR TITLE
TASK: Update yarn command to current format

### PR DESCRIPTION
**Pull request by markusguenther**
Since yarn 1.0 the arguments don`t need the --
So this commit updates the script section to the latest yarn command format.

Fixes: #1545

### CODE REVIEW

**Comment by markusguenther** [comment]
Trigger Travis

File: package.json [hunk 1]
```diff
@@ -7,11 +7,11 @@
     "postinstall": "lerna bootstrap",
     "build": "./Build/createVersionFile.sh && cross-env NODE_ENV=production yarn build:ui",
     "build:ui": "NEOS_BUILD_ROOT=$(pwd) webpack --progress --colors",
-    "build:ui:watch": "yarn build:ui -- --watch",
-    "build:ui:watch-poll": "yarn run build:ui -- --watch-poll --watch",
+    "build:ui:watch": "yarn build:ui watch",
```
Comment (mstruebing) [inline package.json]: `--watch` is needed, otherwise there is an error when running `yarn build:ui:watch`

File: package.json [hunk 2]
```diff
@@ -7,11 +7,11 @@
...
+    "build:ui:watch-poll": "yarn run build:ui watch-poll watch",
```
Comment (mstruebing) [inline package.json]: `yarn run build:ui --watch-poll --watch`

File: package.json [hunk 3]
```diff
@@ -7,11 +7,11 @@
...
     "start-storybook": "lerna run --scope @neos-project/react-ui-components start",
     "test": "lerna run test --concurrency 1",
-    "test:e2e": "yarn run testcafe -- chrome:headless Tests/IntegrationTests/* --selector-timeout=30000 --assertion-timeout=30000",
+    "test:e2e": "yarn run testcafe chrome:headless Tests/IntegrationTests/* selector-timeout=30000 assertion-timeout=30000",
```
Comment (mstruebing) [inline package.json]: `yarn run testcafe chrome:headless Tests/IntegrationTests/* --selector-timeout=30000 --assertion-timeout=30000`

**Comment by markusguenther** [comment]
Thanks @mstruebing

**Pull request merged**